### PR TITLE
US122489 - Wizard Auto versioning

### PR DESCRIPTION
**Pull request by Syeda-Azim**
https://rally1.rallydev.com/#/431372673576d/iterationstatus?detail=%2Fuserstory%2F457631155344&fdp=true

### CODE REVIEW

**Comment by svanherk** [comment]
> Could not generate new goldens - your code changes will update golden files that you do not have the latest version of. Please rebase or merge master into your branch.

This comment is made when there's an issue applying the stash, but the actual problem seems to be mocha complaining it can't find any visual diff test files `Error: No test files found`.

I can try to fix this in the action itself (basically, if there's no tests, visual-diff should pass) but until then you can either wait to setup visual-diff until you have some tests, or write a simple test for now

**Comment by Syeda-Azim** [comment]
> > Could not generate new goldens - your code changes will update golden files that you do not have the latest version of. Please rebase or merge master into your branch.
> 
> This comment is made when there's an issue applying the stash, but the actual problem seems to be mocha complaining it can't find any visual diff test files `Error: No test files found`.
> 
> I can try to fix this in the action itself (basically, if there's no tests, visual-diff should pass) but until then you can either wait to setup visual-diff until you have some tests, or write a simple test for now

Removing visual-diff set up for now, will prefer to wait until the action is fixed.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -20,5 +20,3 @@ jobs:
         uses: BrightspaceUI/actions/semantic-release@master
         with:
           GITHUB_TOKEN: ${{ secrets.D2L_GITHUB_TOKEN }}
-          NPM: true
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (svanherk) [inline .github/workflows/release.yml:24]: You'll want to leave this in this case, we have all our labs components publishing to npm and pull them in that way

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -2,17 +2,24 @@ name: CI
...
+        run: npm run lint:style
+      - name: Lint (Lit)
+        run: npm run lint:lit
+      - name: Unit Tests (SauceLabs)
+        run: npm run test:sauce
         env:
-          SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY_LWEBER }}
-          SAUCE_USERNAME: lweber-d2l
+          SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY_DESIRE2LEARN }}
+          SAUCE_USERNAME: Desire2Learn
```
Comment (svanherk) [inline .github/workflows/ci.yml:25]: I think you'll want to keep using your team's account for Sauce, no?

Comment (Syeda-Azim) [reply .github/workflows/ci.yml:25]: Used 'Desire2Learn' account for `custom-ads-scheduler` repo, so thought of doing the same for wizard.

**Review by bamcdole — APPROVED** [review]

**Review by bamcdole — APPROVED** [review]

**Review by JeremyWinkler — APPROVED** [review]

**Pull request merged**